realm/realm-library/src/main/java/io/realm/OsRealmSchema.java

```diff
@@ -34,7 +34,7 @@
  */
```

Comment:
I don't think you should change this class.  `OsRealmSchema` can not handle generic `RealmObjectSchema`.  It is only able to handle `OsRealmObjectSchema`.  It should use typing to make that clear.